packages/button/src/ButtonBase.tsx

```diff
@@ -11,31 +11,31 @@ function getSizeCss(props: ButtonProps): string {
   switch (props.size) {
```

Comment:
Can we move helper functions below what's getting exported to match the pattern we have on the FE?

Reply:
https://github.com/little-spoon-dev/design-system/pull/36#discussion_r714267022